server/test/services/openapi/v2/Swagger2SchemaGeneratorTest.java

```diff
@@ -0,0 +1,197 @@
+        type: "string"
+      - name: "toDate"
+        in: "query"
+        description: "An ISO-8601 formatted date (i.e. YYYY-MM-DD). Limits results\\
+          \\ to applications submitted on or after the provided date."
+        required: false
+        type: "string"
+      - name: "pageSize"
+        in: "query"
+        description: "A positive integer. Limits the number of results per page. If\\
+          \\ pageSize is larger than CiviForm's maximum page size then the maximum\\
+          \\ will be used. The default maximum is 1,000 and is configurable."
+        required: false
+        type: "integer"
+      - name: "nextPageToken"
+        in: "query"
+        description: "An opaque, alphanumeric identifier for a specific page of results.\\
+          \\ When included CiviForm will return a page of results corresponding to\\
+          \\ the token."
+        required: false
+        type: "string"
+      responses:
+        "200":
+          description: "For valid requests."
+          headers:
+            x-next:
+              type: "string"
+              description: "A link to the next page of responses"
+          schema:
+            $ref: "#/definitions/result"
+        "400":
+          description: "Returned if any request parameters fail validation."
+        "401":
+          description: "Returned if the API key is invalid or does not have access\\
+            \\ to the program."
+securityDefinitions:
+  basicAuth:
+    type: "basic"
+definitions:
+  result:
+    type: "object"
+    properties:
+      payload:
+        type: "array"
+        items:
+          type: "object"
+          properties:
+            applicant_id:
+              type: "integer"
+              format: "int32"
+            application:
+              type: "object"
+            application_id:
+              type: "integer"
+              format: "int32"
+            create_time:
+              type: "string"
+              format: "date-time"
+            language:
+              type: "string"
+            program_name:
+              type: "string"
+            program_version_id:
+              type: "integer"
+              format: "int32"
+            revision_state:
+              type: "string"
+            status:
+              type: "string"
+              x-nullable: true
+            submit_time:
+              type: "string"
+              format: "date-time"
+            submitter_type:
+              type: "string"
+            ti_email:
+              type: "string"
+              x-nullable: true
+            ti_organization:
+              type: "string"
+              x-nullable: true
+      nextPageToken:
+        type: "string"
+""";
+
+    System.out.println("**************************************");
+    System.out.println(actual);
+    System.out.println("**************************************");
+    System.out.println(expected);
+    System.out.println("**************************************");
```

Comment:
debug leftovers?

Reply:
I was going to leave is as the default diff makes it difficult to compare long strings.